sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubConsumer/EventHubConsumerLiveTests.cs

```diff
@@ -693,16 +688,15 @@ await using (var consumer = client.CreateConsumer(EventHubConsumer.DefaultConsum
                             // sent events may not be immediately available.  Allow for a small number of attempts to receive, in order
                             // to account for availability delays.
 
-                            var expectedEventsCount = isInclusive ? 2 : 1;
                             var receivedEvents = new List<EventData>();
                             var index = 0;
 
-                            while ((receivedEvents.Count < expectedEventsCount) && (++index < ReceiveRetryLimit))
+                            while ((receivedEvents.Count < 2) && (++index < ReceiveRetryLimit))
```

Comment:
nit: I think there is still benefit in having `expectedEventsCount` and just setting it to 2.   It keeps things collected in the face of change and offers context to readers.

Reply:
Done!